happi/backends/json_db.py

```diff
@@ -240,31 +228,29 @@ def comparison(name, doc):
 
     def save(self, _id, post, insert=True):
         """
-        Save information to the database
+        Save information to the database.
 
         Parameters
         ----------
         _id : str
-            ID of device
-
+            ID of device.
         post : dict
-            Information to place in database
-
+            Information to place in database.
         insert : bool, optional
-            Whether or not this a new device to the database
+            Whether or not this is a new device to the database.
 
         Raises
         ------
-        DuplicateError:
-            If insert is True, but there is already a device with the provided
-            _id
-
-        SearchError:
-            If insert is False, but there is no device with the provided _id
-
-        PermissionError:
-            If the write operation fails due to issues with permissions
+        DuplicateError
+            If ``insert`` is `True`, but there is already a device with the
+            provided ``_id``.
```

Comment:
Blank lines between exceptions?

Reply:
No, I don't think so. numpydoc isn't explicit about it but none of their examples for other sections have blank lines between.